### PR TITLE
Automatically add sample metadata when importing new bioprojects

### PR DESCRIPTION
**Pull request by lennijusten**
This PR adds a few features:
1. Automatically make the `/bioproject/{bioprojects}/metadata` dir and populate the `metadata.tsv` file with sample-ids when fetching the actual sequencing data. 
2. Fetch the SRA metadata from NCBI using their E-utilities API and save in /bioproject/{bioprojects}/metadata/metadata-ncbi.tsv. 

The "Run" column of the NCBI metadata should match the first column of the `metadata.tsv` but this may not always be the case (since the data is being downloaded from www.ebi.ac.uk?) hence we still use the `metadata.tsv` file. 

Note - I would also like to add a column name row to `metadata.csv` to better document additional metadata we might add. I haven't done this yet is it will require modifying `prepare-dashboard.py` (and maybe other files?).

### CODE REVIEW

**Comment by lennijusten** [comment]
An example `metadata-ncbi.tsv` is attached here (in csv form)
[metadata-ncbi.csv](https://github.com/naobservatory/mgs-pipeline/files/13433628/metadata-ncbi.csv)

File: import-accession.sh [hunk 1]
```diff
@@ -2,16 +2,52 @@
 
 if [ $# -lt 1 ]; then
     echo "Usage: ./import-accession.sh <accessions>"
+    exit 1
```
Comment (jeffkaufman) [inline import-accession.sh:5]: thanks!

File: import-accession.sh [hunk 2]
```diff
@@ -2,16 +2,52 @@
...
+        echo "Run" > "${metadata_dir}/metadata.tsv"
+        echo "$SRA_FILES" | awk '{print $1}' >> "${metadata_dir}/metadata.tsv"
+
+        if check_eutilities; then
+            # Fetch metadata from NCBI
+            echo "Fetching metadata for $ACCESSION from NCBI"
+            METADATA=$(esearch -db sra -query "$ACCESSION" | efetch -format runinfo)
+
+            if [ -n "$METADATA" ] && [ "$(echo "$METADATA" | grep -v '^Run,' | wc -l)" -gt 0 ]; then
+                echo "$METADATA" > "${metadata_dir}/metadata-ncbi.tsv"
```
Comment (jeffkaufman) [inline import-accession.sh:44]: What does NCBI-format metadata look like?

Comment (lennijusten) [reply import-accession.sh:44]: I attached a sample file in an above comment

**Comment by jeffkaufman** [comment]
Can you move this out of draft so it will let me review it?

**Review by jeffkaufman — APPROVED** [review]

**Pull request merged**